### ISSUE TITLE
Order item rows isn´t enabled

### ISSUE THREAD

**Issue by viktormaar** [wetcat-studios/fortie#64]
Is it possible that the `OrderRows` property sanitizes out when posting/putting an order? Or am I doing something wrong?
I can easily fix it, if this issue is confirmed. Only wondering, how nobody ran into this problem before 🤔

**Comment by jongotlin** [comment]
I'm not sure all parts of this api wrapper is tested with real requests. I guess most of it is written from the api documentation and not fully tested and that's why I think you are the first having this issue. 
Please open a pr with the fix.

**Comment by agoransson** [comment]
Yeah, I never actually tested every single request. Most of it is written based on documentation.

**Comment by viktormaar** [comment]
Somehow I presumed that the order export is a commonly used method for the users of this package. :)
Then I´ll fix it soon.
